doc/manual/install-judgehost.rst

```diff
@@ -175,15 +197,15 @@ Starting the judgedaemon
 
 Finally start the judgedaemon::
 
-  bin/judgedaemon -n 0
+  bin/judgedaemon -n 2
 
 Upon its first connection to the domserver API, the judgehost will be
 auto-registered and will be by default enabled. If you wish to
-add a new judgehost but have it initially disabled, you can add it
-manually through the DOMjudge web interface and set it to disabled
-before starting the judgedaemon.
+add a new judgehost but have it initially disabled, you can change the config
+setting to automatically pause judges on first connection or manually add it
+through the DOMjudge web interface and set it to disabled before starting
+the judgedaemon.
 
 The judgedaemon can also be run as a service by running::
 
-  sudo systemctl enable domjudge-judgehost
-  sudo systemctl start  domjudge-judgehost
+  sudo systemctl enable --now domjudge-judgehost
```

Comment:
Should this be `systemctl start`?

Reply:
`enable --now` is shorthand for `systemctl {enable,start}`, (or it was).

`enable` creates the symlink, the --now is telling that you also want to directly start.